software/nginx/en.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "nginx"
+lastmod = "2020-04-03T23:40:51+02:00"
++++
+# nginx
+Solus provides the nginx web server and proxy to users and developers who want to test their web applications before setting them to production. \
```

Comment:
What's up with the \ ?

Reply:
It's for a line break.

Reply:
Please no. Just use new lines.

Reply:
Pure newlines also add a new paragraph that I didn't mean to add, but OK! I'll fix all the issues by the end of the day.